apps/docs/context/theme/getInitialColorMode.ts

```diff
@@ -2,12 +2,12 @@ import type { ColorScheme } from "./ThemeProvider";
 
 export function getInitialColorMode(): ColorScheme {
     const persistedColorPreference = window.localStorage.getItem("hdTheme");
-    const hasPersistedPreference = typeof persistedColorPreference === "string";
+    const hasPersistedPreference = true;
```

Comment:
This seems odd. it'll always be true?

Reply:
Well seen, this change was made quickly between several others.